src/geometry/euclidean/basic.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2020 Joseph Myers. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Joseph Myers, Manuel Candales
 -/
-import analysis.normed_space.inner_product
+import analysis.normed_space.pi_Lp
```

Comment:
As far as I can tell, the `euclidean_space` construction is never used in `geometry/euclidean`; the model space seems to be always a generic inner product space.  Can you remove the dependency?